web/src/components/table/use-cases/sessions.tsx

```diff
@@ -282,6 +302,7 @@ export default function SessionsTable({
       id: "usage",
       header: "Usage",
       enableHiding: true,
+      enableSorting: true,
```

Comment:
Same as traces table. Does this work?

Reply:
Yes it does. Usage column behaves internally like total_token column. As reference the sessionTable.ts defintion:
`  {
    name: "Usage",
    id: "usage",
    type: "number",
    internal: 'o."totalTokens"',
  },`